lib/ansible/modules/system/aix_lvg.py

```diff
@@ -0,0 +1,275 @@
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+author: Kairo Araujo (@kairoaraujo)
+module: aix_lvg
+short_description: Configure LVM volume groups for AIX.
+description:
+  - This module creates, removes or resize volume groups on AIX LVM.
+version_added: "2.5"
+options:
+  vg:
+    description:
+      - Volume group name.
+    required: true
+  pvs:
+    description:
+      - List of comma-separated devices to use as physical devices in this
+        volume group. Required when creating or extending (C(present) state)
+        the volume group. If not informed reducing (C(absent) state) the volume
+        group will be removed.
+    required: false
+  pp_size:
+    description:
+      - Size of the physical partition in megabytes.
+    choices: [1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024]
+    default: 32
+    required: false
+  vg_type:
+    description:
+      - Volume group type.
+    choices: [normal, big, scalable]
+    default: normal
+    required: false
+  state:
+    description:
+      - Control if the volume group exists.
+    choices: [ present, absent ]
+    default: present
+    required: false
+  force:
+    description:
+      - Forces volume group creation.
+    choices: ["yes", "no"]
+    default: "no"
+notes:
+  - AIX will permit remove VG only if all LV/Filesystems aren't busy.
+  - Module does not modify PP size for already present volume group.
+'''
+
+EXAMPLES = '''
+- name: Create a volume group datavg
+  aix_lvg:
+    vg: datavg
+    pp_size: 128
+    vg_type: scalable
+    state: present
+
+- name: Removing a volume group datavg
+  aix_lvg:
+    vg: datavg
+    state: absent
+
+- name: Extending rootvg
+  aix_lvg:
+    vg: rootvg
+    pvs: hdisk1
+    state: present
+
+- name: Reducing rootvg
+  aix_lvg:
+    vg: rootvg
+    pvs: hdisk1
+    state: absent
+'''
+
+RETURN = '''
+changed:
+  description: Return changed for aix_lvg actions as true or false.
+  returned: always
+  type: boolean
+  version_added: 2.5
+msg:
+    description: Return message regarding the action.
+    returned: always
+    type: string
+    version_added: 2.5
+'''
+
+
+from ansible.module_utils.basic import AnsibleModule
+
+# vg mode parameter
+vg_mode = {
+    'normal': '',
+    'big': '-B',
+    'scalable': '-S',
+}
+
+# force parameter
+force_mode = {
+    True: '-f',
+    False: ''
+}
+
+
+def _validate_pv(module, pvname):
+    """
+    Function to validate if the physical volume (PV) is not already in use by
+    another volume group or Oracle ASM
+
+    :param module: Ansible module argument spec
+    :param pvname: Physical Volume name
+    :return: [True, message]
+    """
+
+    lspv_cmd = module.get_bin_path('lspv', True)
+    rc, current_lspv, err = module.run_command('%s ' % lspv_cmd)
```

Comment:
Regarding single/double quotes used across the code, there is non-written convention to use double quotes only for text with placeholders (e.g. `"%s "`) or longer text messages (e.g. `msg="Failed executing lspv command."`). When a text contains double quotes, to avoid escaping the quotes, single quotes should be used (e.g. `my_var = 'This is "my" variable.'`). The rest of cases should use single quotes. Please walk through your code and unify the quotes according this convention.

Reply:
indeed

Reply:
I see did some changes of single quotes to double quotes but you left all double quotes are the were. You should change all other cases than named above to single quotes (e.g. keys in the dict vars - `my_dict['my_key']`).